mmv1/third_party/terraform/services/compute/resource_compute_network_test.go.erb

```diff
@@ -22,14 +22,16 @@ func TestAccComputeNetwork_explicitAutoSubnet(t *testing.T) {
 	t.Parallel()
 
 	var network compute.Network
+	suffixName := acctest.RandString(t, 10)
+    networkName := fmt.Sprintf("tf-test-network-basic-%s", suffixName)
```

Comment:
```suggestion
	suffixName := acctest.RandString(t, 10)
	networkName := fmt.Sprintf("tf-test-network-basic-%s", suffixName)
```

Could you please make this type of adjustment for all the affected tests? Thanks!